### DISCUSSION TITLE
Change port in MiniO service and some others bugs

### DISCUSSION THREAD

**Discussion by sawa-ko** [coollabsio/coolify#559] [Q&A] [answered]
When I install the MiniO service I see that it is always on port 9000, and I don't understand why it can't be changed.

Also, I see that the MINIO_SERVER_URL environment variable is not configured correctly because if you try to share a file, it has an IP address as a URL and not the API URL (http://172.16.16.10:9000/file.ext). I try to do it manually, adding a secret to the service with that name, and it seems to work, however, it is no longer possible for me to log in, it always appears invalid session. That happens even though i literally copy and paste the login data.

**Comment by peaklabs-dev** [comment, marked as answer]
This has been added in v4.
